src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -1274,23 +1274,23 @@ class CommonVisitor : public AST::BaseVisitor<Struct> {
                 return ASRUtils::make_SubroutineCall_t_util(al, loc, stemp,
                     s_generic, args_new.p, args_new.size(), nullptr, nullptr, false, false);
             }
-        } else if(ASR::is_a<ASR::StructType_t>(*s)) {
-            ASR::StructType_t* StructType = ASR::down_cast<ASR::StructType_t>(s);
+        } else if(ASR::is_a<ASR::Struct_t>(*s)) {
+            ASR::Struct_t* Struct = ASR::down_cast<ASR::Struct_t>(s);
```

Comment:
```suggestion
            ASR::Struct_t* struct = ASR::down_cast<ASR::Struct_t>(s);
```
Minor

Reply:
This changes needs to be updated wherever it is used.

Reply:
Making this lower case causes conflicts with the `struct` keyword.

Reply:
This is fixed now.